wireguard-install.sh

```diff
@@ -121,28 +136,47 @@ function installWireGuard() {
 	# Install WireGuard tools and module
 	if [[ ${OS} == 'ubuntu' ]]; then
 		apt-get update
-		apt-get install -y wireguard iptables resolvconf qrencode
+		if [[ $Container == 1 ]]; then
+			apt-get install -y wireguard-tools --no-install-recommends
+			apt-get install -y iptables resolvconf qrencode
+		else
+			apt-get install -y wireguard iptables resolvconf qrencode
+		fi
+
```

Comment:
Install only wireguard-tools for Ubuntu, as that is all that is needed when Kernel Module works.

Reply:
I would write it like this to make the differences clearer and avoid duplication:

```suggestion
		apt-get install -y iptables resolvconf qrencode
		if [[ $Container == 1 ]]; then
			apt-get install -y wireguard-tools --no-install-recommends
		else
			apt-get install -y wireguard
		fi
```

Same of the other distros